urbanaccess/gtfs/utils_format.py

```diff
@@ -441,15 +457,25 @@ def add_unique_agencyid(agency_df=None,stops_df=None,routes_df=None,trips_df=Non
                                                 trips_df=trips_df[['trip_id', 'route_id']],
                                                 agency_df=agency_df[['agency_id','agency_name']])
 
+            # TODO: It's obfuscatory to update the dataframe variables in such a deeply nested way.
```

Comment:
Yes agreed! Your changes are more clear. Please remove this TODO from this PR since if you dont mind as you have fixed the issue.